controllers/idler/idler_controller.go

```diff
@@ -132,6 +132,7 @@ func (r *Reconciler) ensureIdling(ctx context.Context, idler *toolchainv1alpha1.
 	}
 	newStatusPods := make([]toolchainv1alpha1.Pod, 0, 10)
 	for _, pod := range podList.Items {
+		pod := pod
```

Comment:
This is fixed in go 1.22 btw: https://go.dev/blog/loopvar-preview
So maybe worth adding a comment?
Something like:
```suggestion
		pod := pod // TODO We won't need it after upgrading to go 1.22: https://go.dev/blog/loopvar-preview
```

Reply:
done in 74e1f570dbc3d4dac759330185c0e764390d0ff1